<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Feed</class>
 <widget class="QDialog" name="Feed">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="logout_btn">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>0</y>
     <width>71</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Logout</string>
   </property>
  </widget>
  <widget class="QPushButton" name="feed_btn">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>0</y>
     <width>100</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>Feed</string>
   </property>
  </widget>
  <widget class="QPushButton" name="notifications_bttn">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>0</y>
     <width>95</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Notifications</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget_users_images">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>40</y>
     <width>171</width>
     <height>291</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_view_2">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>340</y>
     <width>114</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>View image</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget_users">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>171</width>
     <height>291</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="profile_btn">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>81</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Profile</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_view">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>83</x>
     <y>340</y>
     <width>121</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>View all images</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_red_mark">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>0</y>
     <width>15</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="search_btn">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>0</y>
     <width>81</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
  </widget>
  <zorder>logout_btn</zorder>
  <zorder>listWidget_users_images</zorder>
  <zorder>pushButton_view_2</zorder>
  <zorder>listWidget_users</zorder>
  <zorder>profile_btn</zorder>
  <zorder>pushButton_view</zorder>
  <zorder>search_btn</zorder>
  <zorder>feed_btn</zorder>
  <zorder>notifications_bttn</zorder>
  <zorder>label_red_mark</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
